src/main/java/com/conveyal/r5/streets/StreetRouter.java

```diff
@@ -337,7 +337,7 @@ public boolean setOrigin (double lat, double lon) {
         State startState0 = new State(split.vertex0, split.edge + 1, streetMode);
         State startState1 = new State(split.vertex1, split.edge, streetMode);
         EdgeStore.Edge  edge = streetLayer.edgeStore.getCursor(split.edge);
-        int offStreetTime = split.distanceToEdge_mm / OFF_STREET_SPEED_MILLIMETERS_PER_SECOND;
+        int offStreetTime = split.distanceToEdge_mm / (int) (profileRequest.walkSpeed * 1000);
```

Comment:
It may make more sense to perform floating point multiplication and division and only convert the result to int before storing it in offStreetTime. But integer division was already being used on mm-per-second speeds in related code in LinkedPointSet so this consistent, and rethinking the use of integer and float operations is not within scope for this PR.